src/components/calculator/calculator.ts

```diff
@@ -22,6 +22,7 @@ interface IResourceItem {
 }
 
 interface ICalculatorState {
+  monthEstimateIsValid: string;
```

Comment:
This variable name is a little confusing. I initially read it as a boolean "is the 'month estimate' valid", but I now see it's supposed to be "the month and year this estimate is valid for". Perhaps just appending `monthEstimateIsValidFor` would be enough to prevent people from misreading it as a boolean? Or `monthOfEstimate`?

Reply:
`monthOfEstimate` is better, I have amended.